Sources/MapboxNavigation/Resources/en.lproj/Localizable.strings

```diff
@@ -226,6 +238,18 @@
 /* Specific route feedback that route contained impassible, narrow roads. */
 "ROUTE_QUALITY_ROADS_TOO_NARROW_FEEDBACK" = "Route had roads too narrow to pass";
 
+/* Alternatives selection note about equal travel distance. */
+"SAME_DISTANCE" = "Same distance";
+
+/* Alternatives selection note about equal travel time. */
+"SAME_TIME" = "Same time";
+
+/* Alternatives selection note about a shorter route. */
```

Comment:
Document that the placeholder contains the difference in time, which may be in any unit.